modules/client/src/default.ts

```diff
@@ -1,16 +1,21 @@
 import { ConnextStore } from "@connext/store";
-import { ClientOptions, StoreTypes, IClientStore } from "@connext/types";
-import { getRandomPrivateKey, removeUndefinedFields } from "@connext/utils";
+import { ClientOptions, StoreTypes } from "@connext/types";
+import { hexlify, randomBytes } from "ethers/utils";
 
 const CONNEXT_DEFAULT_SIGNER_KEY = "CONNEXT_DEFAULT_SIGNER";
 
+const removeUndefinedFields = <T>(obj: T): T => {
+  Object.keys(obj).forEach(key => typeof obj[key] === "undefined" && delete obj[key]);
+  return obj;
+};
+
 const getGeneratedSigner = (): string => {
   if (typeof window === "undefined" || typeof window.localStorage === "undefined") {
-    return getRandomPrivateKey();
+    return hexlify(randomBytes(32));
```

Comment:
shouldn't this use getRandomBytes32 instead

Reply:
Yea, this particular `hexlify(randomBytes` call was temporary to fix broken imports while reorganizing. I replaced it with a proper getRandom function in a later commit :+1:

Reply:
Oh wait, this was in a different spot than I thought. No I didn't fix it but I will. :facepalm: